operator-sdk script is broken on macos local dev env

**Steps to reproduce**

On macos:

①

```sh
rm -rf ./bin/operator-sdk
```

②

```sh
make operator-sdk
# ./utils/install-operator-sdk.sh /Users/guilherme.cassolato/Workspace/go/src/github.com/kuadrant/kuadrant-operator/bin/operator-sdk v1.28.1
# mkdir: /Users/runner: Permission denied
# make: *** [/Users/guilherme.cassolato/Workspace/go/src/github.com/kuadrant/kuadrant-operator/bin/operator-sdk] Error 1
```

<br/>

Introduced with https://github.com/Kuadrant/kuadrant-operator/pull/268/commits/72f10e34f1af5de9c9f0de9d5ba2c5875a578a88

--- a/utils/install-operator-sdk.sh
+++ b/utils/install-operator-sdk.sh
@@ -16,9 +16,9 @@ if [ ! -f "$1" ]; then
 
   # GPG can hit "no route for host" if using IPv6 on mac (https://github.com/rvm/rvm/issues/4215#issuecomment-486609350)
   if [[ $OS == 'darwin' ]]; then
-    mkdir -p "/Users/runner/.gnupg"
+    mkdir -p "$HOME/.gnupg"
     echo "Disabling ipv6 for gpg on mac"
-    echo "disable-ipv6" >  /Users/runner/.gnupg/dirmngr.conf
+    echo "disable-ipv6" > $HOME/.gnupg/dirmngr.conf
     gpgconf --kill all
   fi
 

